specification/syntex/resource-manager/Microsoft.Syntex/preview/2022-09-15-preview/syntex.json

```diff
@@ -0,0 +1,521 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Operations_List": {
+            "$ref": "./examples/Operations_List.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DocumentProcessorResource": {
+      "type": "object",
+      "description": "Document processor details",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true
+        },
+        "properties": {
+          "x-ms-client-flatten": false,
+          "description": "Document processor properties.",
+          "$ref": "#/definitions/DocumentProcessorProperties"
+        }
+      }
+    },
+    "DocumentProcessorPatchableProperties": {
+      "type": "object",
+      "description": "Document processor properties that can be patched",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "Resource tags.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "description": "Syntex properties for PATCH.",
+          "properties": {
+            "spoTenantUrl": {
+              "type": "string",
+              "description": "The URL of an SPO tenant associated with this document processor resource"
+            },
+            "spoTenantId": {
+              "type": "string",
+              "description": "The Id (GUID) of an SPO tenant associated with this document processor resource"
+            }
+          }
+        }
+      }
+    },
+    "DocumentProcessorProperties": {
+      "type": "object",
+      "description": "Document processor properties",
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The managed resource provisioning state."
+        },
+        "spoTenantUrl": {
+          "type": "string",
+          "description": "The URL of an SPO tenant associated with this document processor resource"
+        },
+        "spoTenantId": {
+          "type": "string",
+          "description": "The Id (GUID) of an SPO tenant associated with this document processor resource"
+        }
+      },
+      "required": [
+        "spoTenantUrl",
+        "spoTenantId"
+      ]
+    },
+    "ProvisioningState": {
+      "description": "Provisioning status of the managed resource.",
+      "readOnly": true,
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled"
```

Comment:
Do you have any long-running create or update operations during which you will return additional non-terminal states? What states?

Reply:
No, we don't have any other long-running operations.  We discussed this at the ARM review board meeting earlier this year.